e2e/pages/adf/dataTablePage.ts

```diff
@@ -44,20 +45,107 @@ export class DataTablePage {
         this.list = this.rootElement.all(by.css(`div[class*=adf-datatable-body] div[class*=adf-datatable-row]`));
     }
 
+    clickRowToSelectWithRoot(rowName) {
+        let row = this.getRowByRowNameWithRoot(rowName);
+        browser.actions().keyDown(protractor.Key.COMMAND).click(row).perform();
+        this.checkRowIsSelectedWithRoot(rowName);
+        return this;
+    }
+
+    checkRowIsSelectedWithRoot(content) {
+        let isRowSelected = this.getRowsNameWithRoot(content).element(by.xpath(`ancestor::div[contains(@class, 'is-selected')]`));
+        Util.waitUntilElementIsVisible(isRowSelected);
+    }
+
+    getRowsNameWithRoot(content) {
+        let row = this.list.all(by.css(`adf-datatable span[title='${content}']`)).first();
+        Util.waitUntilElementIsVisible(row);
+        return row;
+    }
+
+    getRowByRowNameWithRoot(content) {
+        let rowByRowName = by.xpath(`ancestor::div[contains(@class, 'adf-datatable-row')]`);
+        Util.waitUntilElementIsVisible(this.getRowsNameWithRoot(content).element(rowByRowName));
+        return this.getRowsNameWithRoot(content).element(rowByRowName);
+    }
+
+    getColumnValueForRow(file, columnName) {
+        let row = this.getRowByRowName(file);
+        Util.waitUntilElementIsVisible(row);
+        let rowColumn = row.element(by.css(`div[title="${columnName}"] span`));
+        Util.waitUntilElementIsVisible(rowColumn);
+        return rowColumn.getText();
+    }
+
+    async getStyleValueForRowText(rowName, styleName) {
```

Comment:
I don't see this method used in the project. If this is the case, please remove it.